app/models/candidate.rb

```diff
@@ -25,10 +28,16 @@ def prepare(user)
       self.stil_id = user.profile.stil_id
     end
   end
+
+  def owner?(user)
+    user.present? && user.profile == profile
+  end
+
   def editable?
-    return (self.election.view_status == 2) || (self.post.elected_by == "Studierådet")
+    election.view_status == 2 || post.elected_by == "Studierådet"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.